pages/home.py

```diff
@@ -35,69 +37,188 @@ def get_countries(economies: Economies) -> list[str]:
 
 
 def get_counterpart_countries(economies: Economies, country: str) -> list[str]:
-    'Get counterpart country list based on country'
-    df = (
-        dfc_imf_dot
-        .query(f"`Country Name` == '{country}'")['Counterpart Country Name']
-        .unique())
+    "Get counterpart country list based on country"
+    df = dfc_imf_dot.query(f"`Country Name` == '{country}'")[
+        "Counterpart Country Name"
+    ].unique()
```

Comment:
these new lines are making it harder to read. Try to follow pep8 and put newline before each chained function call to make it easier to read:
```python
df = (dfc_imf_dot
    .query(f"`Country Name` == '{country}'")["Counterpart Country Name"]
    .unique())
```
This way I can see right away this line called 2 functions. Especially helpful for longer chains.